src/sql/workbench/api/common/extHostNotebookDocumentData.ts

```diff
@@ -50,6 +49,9 @@ export class ExtHostNotebookDocumentData implements IDisposable {
 				get cells() { return data._cells; },
 				get kernelSpec() { return data._kernelSpec; },
 				save() { return data._save(); },
+				// ignore error "setTrusted' does not exist in type 'NotebookDocument'." as it is merged from azdata.proposed.d.ts
+				// @ts-ignore
+				setTrusted(isTrusted) { data._setTrusted(isTrusted); },
```

Comment:
@Charles-Gagnon I thought we merged azdata and the proposed version at compile time? Am I misremembering something?

Reply:
Correct. Already moved it to proposed, but forgot to remove the comment. Thanks!